tests/ConversationTranslatorTests.ts

```diff
@@ -84,6 +86,401 @@ describe("conversation constructor tests", () => {
+        if (endpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Host], endpointHost); }
+        if (speechEndpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+        config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Name], "Host.Testing");
+
+        const c = sdk.Conversation.createConversationAsync(config, undefined, ((error: any) => {
+            expect(error).toContain("400025");
+            errorMessage = error;
+        }));
+        objsToClose.push(c);
+
+        WaitForCondition(() => (errorMessage !== undefined), done);
+
+    }, 20000);
+
+    test("Start Conversation, invalid subscription key or region [401000]", (done: jest.DoneCallback) => {
+
+        let errorMessage: string;
+
+        const config = sdk.SpeechTranslationConfig.fromSubscription("abc", "def");
+        config.speechRecognitionLanguage = "en-US";
+        if (endpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Host], endpointHost); }
+        if (speechEndpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+        const c = sdk.Conversation.createConversationAsync(config, undefined, ((error: any) => {
+            expect(error).toContain("401000");
+            errorMessage = error;
+        }));
+        objsToClose.push(c);
+
+        WaitForCondition(() => (errorMessage !== undefined), done);
+
+    }, 20000);
+
+    test("Start Conversation, join as host and mute participants", (done: jest.DoneCallback) => {
+
+        // tslint:disable-next-line:no-console
+        console.info("Start Conversation, join as host, mute participants");
+
+        let commandCount: number = 0;
+        let participantsCount: number = 0;
+        let isMuted: boolean = true;
+
+        // start a conversation
+        const config = sdk.SpeechTranslationConfig.fromSubscription(Settings.SpeechSubscriptionKey, Settings.SpeechRegion);
+        if (endpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.ConversationTranslator_Host], endpointHost); }
+        if (speechEndpointHost !== "") { config.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+        const c: sdk.Conversation = sdk.Conversation.createConversationAsync(config, (() => {
+            const ct: sdk.ConversationTranslator = new sdk.ConversationTranslator();
+            if (endpointHost !== "") { ct.properties.setProperty(sdk.PropertyId.ConversationTranslator_Host, endpointHost); }
+            if (speechEndpointHost !== "") { ct.properties.setProperty(sdk.PropertyId[sdk.PropertyId.SpeechServiceConnection_Host], speechEndpointHost); }
+
+            ct.canceled = ((s: sdk.ConversationTranslator, e: sdk.ConversationTranslationCanceledEventArgs) => {
+                done();
```

Comment:
>  done(); [](start = 14, length = 9)

Seems this should check for an error and not just finish with success because canceled.

Reply:
It's checking now for a failure.